app/templates/static/maintaining.static.pug

```diff
@@ -0,0 +1,33 @@
+block content
+  body(style="background-image: url(/images/pages/maintaining/background.jpg); background-size: 100vw; display: flex; flex-direction: column; align-items: center;", onload="onLoad()")
+    img(id='logo', src="/images/pages/base/logo.png")
+    h3(id='title', style="margin-top: 20vh;font-size:48px;color: #317EAC")
+    span(id='msg', style="margin-top: 30vh;font-size:24px;")
+
+block script-start
+  script(src='/maintaining_time')
+  script.
+    function onLoad() {
+      time = maintainingHours
+      duration = null
+      if(!isNaN(+time))
+        duration = +time
```

Comment:
This is a bit confusing, maybe at least a comment saying how `maintenanceHours` is either `"0"` for indefinite maintenance or some string representing the number of hours it is expected to take?